docs/09-英文/01-Introduction/03whatis.md

```diff
@@ -0,0 +1,160 @@
+
+### **Consumer**
+
+Consumers read data by reading messages from the topics to which they subscribe.
+
+- Supports message consumption in two modes: push and pull.
+- Supports **cluster mode** and broadcast mode consumption
+- Provide real-time message subscription mechanism
+
+##  **NameServer**
+
+NameServer is a simple Topic routing registry that supports dynamic registration and discovery of Topic and Broker.
+
+Mainly includes two functions：
+- **Broker management**:The NameServer accepts the registration information of the Broker cluster and saves it as the basic data of the routing information.And it provides a heartbeat detection mechanism to check whether the Broker is still alive.
+- **Routing Information Management**:Each NameServer will hold the entire routing information about the Broker cluster and queue information for client queries.The Producer and Consumer can know the routing information of the entire Broker cluster through the NameServer, so as to produce and consume messages.
+
+NameServer usually has multiple instances deployed, and each instance does not communicate with each other.Broker registers its own routing information with each NameServer, so each NameServer instance saves a complete routing information.The client can still obtain routing information from other NameServers, When a NameServer goes offline for some reason.
+
+##  Broker
+
+Broker is mainly responsible for message storage, delivery and query, as well as service high availability guarantee.
+
+NameServer is stateless, so it can be deployed in clusters without any information synchronization between nodes.Compared with nameserver, broker is more complicated.
+
+In the Master-Slave architecture, the Broker is divided into Master and Slave.
+A Master can correspond to multiple Slaves, but a Slave can only correspond to one Master.
+The correspondence between Master and Slave is defined by specifying the same BrokerName and different BrokerId. A BrokerId of 0 means Master, and non-0 means Slave.Master can also deploy multiple.
+
+
+
+:::note RocketMQ Architecture Summary
+
+- Each **Broker** establishes long-term connections with all nodes in the **NameServer** cluster, and regularly registers Topic information to all NameServers.
+- **Producer** establishes a persistent connection with one of the nodes in the **NameServer** cluster, regularly obtains topic routing information from NameServer, establishes a persistent connection to the Master that provides Topic services, and regularly sends a heartbeat to the Master.Producers are completely stateless.
+- **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster
+，regularly obtains Topic routing information from NameServer,establishes long connections to Master and Slave that provide Topic services, and send heartbeats to Master and Slave regularly.Consumer subscribes to topic from Master or Slave.
```

Comment:
> - **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster
，regularly obtains Topic routing information from NameServer,establishes long connections to Master and Slave that provide Topic services, and send heartbeats to Master and Slave regularly.Consumer subscribes to topic from Master or Slave.

to:

> - **Consumer** establishes a persistent connection to one of the nodes in the **NameServer** cluster, regularly obtains Topic routing information from NameServer, establishes long connections to Master and Slave that provide Topic services, and sends heartbeats to Master and Slave regularly. A Consumer can subscribe to Topics from either Master or Slave.